contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.3.0--3.4.0.sql

```diff
@@ -2560,6 +2584,10 @@ DROP PROCEDURE sys.babelfish_update_user_catalog_for_guest_schema();
 -- Please have this be one of the last statements executed in this upgrade script.
 DROP PROCEDURE sys.babelfish_drop_deprecated_object(varchar, varchar, varchar);
 
+-- After upgrade, always run analyze for all babelfish catalogs.
+CALL sys.analyze_babelfish_catalogs();
```

Comment:
is there way to validate from stats/explain that we have done analyze on catalog tables or not? if there is, we should add a upgrade test to validate the same.

Reply:
added in next commit.